packages/core/src/util/stackTrace.js

```diff
@@ -37,11 +37,17 @@ exports.captureStackTrace = function captureStackTrace(length, referenceFunction
     // drawback is that it might show too much, but that's better than not having any stack trace at all.
     Error.captureStackTrace(stackTraceTarget);
   }
-  const stack = stackTraceTarget.stack;
+
+  let stack = stackTraceTarget.stack;
+
+  if (!Array.isArray(stack)) {
+    stack = [];
+  }
+
   Error.stackTraceLimit = originalLimit;
   Error.prepareStackTrace = originalPrepareStackTrace;
 
-  if (drop > 0) {
+  if (drop > 0 && stack.length >= drop) {
```

Comment:
_praise:_ good catch, that way we at least get some stack trace if `drop > stack.length` accidentally. (I don't think this should ever happen, but anyway.)

Reply:
Yeah 👍